ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -0,0 +1,49 @@
+import tempfile
+import argparse
+import logging
+import os
+
+from ocs_ci.framework import config
+from ocs_ci.ocs.parallel import parallel
+from ocs_ci.ocs.constants import CLEANUP_YAML, TEMPLATE_CLEANUP_DIR
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.utility import templating
+
+logger = logging.getLogger(__name__)
+
+def cleanup(cluster_name, cluster_id):
+    """
+        Cleanup existing cluster in AWS
```

Comment:
```suggestion
    Cleanup existing cluster in AWS
```

Reply:
This still isn't aligned correctly. There should also be an empty line at the end of the docstring.

```
    """
    Cleanup existing cluster in AWS

    Args:
        cluster_name (str): Name of the cluster
        cluster_id (str): Cluster id to cleanup

    """
```

Reply:
Sorry my force pushed messed up previous fix.